server/src/main/java/org/tctalent/server/service/db/impl/SavedListServiceImpl.java

```
@@ -616,6 +621,19 @@ public void mergeSavedListFromInputStream(long savedListId, InputStream is)
         mergeSavedList(savedListId, request);
     }
 
+    private boolean isCandidateNumber(String s) {
+        try {
+            Long.parseLong(s);
+            return true;
+        } catch (NumberFormatException ex) {
```

Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Exception handling issues detected where catch blocks only return values, log messages, or record metrics without properly handling the error. This can cause silent failures where errors go unnoticed, leading to incorrect results and making it hard to debug problems in production. To fix this, make sure catch blocks include proper logging with the full exception object, re-throw exceptions to let callers handle them, or add fallback mechanisms like backup services. Learn more about proper exception handling at https://owasp.org/www-community/Improper_Error_Handling